plugins/filesystem/local/src/Adapter/LocalAdapter.php

```diff
@@ -267,22 +318,31 @@ public function updateFile(string $name, string $path, $data)
      * @since   4.0.0
      * @throws  \Exception
      */
-    public function delete(string $path)
+    public function delete(string $path): void
```

Comment:
```suggestion
    public function delete(string $path)
```
Just to be on the safe side, when somebody extends the LocalAdapter delete function.